web/thesauruses/javascript/ECMAScript 2009/operators.json

```diff
@@ -0,0 +1,220 @@
+{
+    "meta": {
+      "language": "javascript",
+      "language_name": "JavaScript",
+      "structure": "operators",
+      "language_version": "ECMAScript 2009"
+    },
+    "concepts": {
+      "addition": {
+        "name": "Addition operator",
+        "code": "a + b"
+      },
+      "addition_assignment": {
+        "name": "Addition and assignment operator",
+        "code": "a += b"
+      },
+      "subtraction": {
+        "name": "Subtraction operator",
+        "code": "a - b"
+      },
+      "subtraction_assignment": {
+        "name": "Subtraction and assignment operator",
+        "code": "a -= b"
+      },
+      "multiplication": {
+        "name": "Multiplication operator",
+        "code": "a * b"
+      },
+      "multiplication_assignment": {
+        "name": "Multiplication and assignment operator",
+        "code": "a *= b"
+      },
+      "division": {
+        "name": "Division operator",
+        "code": "a / b"
+      },
+      "division_assignment": {
+        "name": "Division and assignment operator",
+        "code": "a /= b"
+      },
+      "integer_division": {
+        "name": "Integer division operator",
+        "code": "a // b"
+      },
+      "integer_division_assignment": {
+        "name": "Integer division and assignment operator",
+        "code": "a //= b"
+      },
+      "modulus": {
+        "name": "Modulus (remainder) operator",
+        "code": "a % b"
+      },
+      "modulus_assignment": {
+        "name": "Modulus and assignment operator",
+        "code": "a %= b"
+      },
+      "unary_plus": {
+        "name": "Unary plus operator",
+        "code": "+a"
+      },
+      "unary_minus": {
+        "name": "Unary minus operator",
+        "code": "-a"
+      },
+      "increment": {
+        "name": "Increment (add 1) operator",
+        "code": [
+            "a++",
+            "++a"
+        ]
+      },
+      "decrement": {
+        "name": "Decrement (subtract 1) operator",
+        "code": [
+            "a--",
+            "--a"
+        ]
+      },
+      "exponential": {
+        "name": "Exponential operator",
+        "code": "a ** b"
+      },
+      "factorial": {
+        "not-implemented": true,
+        "name": "Factorial operator"
+      },
+      "absolute_value": {
+        "name": "Absolute value operator",
+        "code": "Math.abs(a)"
+      },
+      "percentage": {
+        "not-implemented": true,
+        "name": "Percentage operator"
+      },
+      "equal_to": {
+        "name": "Equality operator",
+        "code": [
+          "a == b // loose equality",
+          "a === b // strict equality"
+        ],
+        "comment": "The `===` operator coerces the operands to the same type before comparing them. The `==` operator does not coerce the operands."
```

Comment:
Isn't this backward? If `===` is strict equality, it _doesn't_ coerce the types.

Reply:
Oh. sorry about that, I'll change the comment by the description on MDN about strict equality:
> The `===` operator checks whether its two operands are equal, returning a Boolean. Unlike the `==` operator, the strict equality operator always considers operands of distinct types to be different.

References :
- MDN page about strict equality: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality